crates/sui-json-rpc-types/src/sui_object.rs

```diff
@@ -770,8 +770,39 @@ impl SuiData for SuiParsedData {
     }
 
     fn try_from_package(package: MovePackage) -> Result<Self, anyhow::Error> {
+        fn disassemble_modules<'a, I>(modules: I) -> SuiResult<BTreeMap<String, Value>>
```

Comment:
At this point you probably don't need to wrap this in a function

Reply:
true...i was just moving code though, i'll fix this before landing